addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -315,54 +318,81 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend({
             resolveFadeOut();
         });
 
-        var nextQuestionPromise = this._rpc({
-            route: _.str.sprintf('/survey/session/next_question/%s', self.surveyAccessToken),
-            params: {
-                'go_back': goBack,
-            }
-        });
+        var refreshBackground = this.reloadBackgroundOnNext || goBack;
```

Comment:
Mmh "refreshBackground" sounds more like a method than a condition.
Not sure what would fit best, "needBackgroundRefresh" maybe?